Gemfile

```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+ruby RUBY_VERSION
+
+gem 'bundler'
```

Comment:
sorry @olleolleolle, GH hid these comments from me behind an "outdated diff", and I just thought to check. I do understand about the Gemfile vs. .gemspec for a gem. I'm [reading & trying to learn](https://github.com/bundler/bundler/pull/4651) more about this issue. I work on a project where this *could* be very helpful in our tests...